src/main/scripts/record-extract.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+#
+# This script mimics daily_extract.sh but works on specific records, rather than all in a daily export.
+# This is for use in recovery, when specific records need to be uploaded to LARM
+#
+
+check_parameters() {
+    if [[ -z "$IDFILE" ]]; then
```

Comment:
`-s` checks that the file exists and is not empty and seems a better fit that `-z`.

Reply:
Or maybe `-f` if zero-length files are ok.